src/templates/chatbox_head.html

```diff
@@ -3,11 +3,11 @@
     <div class="chatbox-title">
         <div class="row no-gutters">
             {[ if (o.type !== o._converse.HEADLINES_TYPE) { ]}
-                <canvas class="avatar" height="36" width="36"></canvas>
+               <div style="cursor:pointer;"></a> <canvas class="avatar show-user-details-modal" height="36" width="36"></canvas></div>
```

Comment:
Please don't put `style` attributes in the markup. All styles should go in the Sass/CSS.